doc/faq.rst

```diff
@@ -282,6 +282,33 @@ usually a major and lengthy undertaking, it is recommended to start with
 :ref:`known issues <new_contributors>`. Please do not contact the contributors
 of scikit-learn directly regarding contributing to scikit-learn.
 
+How can I be active within the community?
```

Comment:
Could you remove this part.

The original issue only mentioned to add a link in the front page. Let's not add anything in the FAQ.